site/src/app/v2/test-eligibilite/components/step-one-form/StepOneForm.tsx

```diff
@@ -76,7 +77,12 @@ const StepOneForm = ({ onDataReceived }: Props) => {
           notifyError(status, body);
           return;
         }
+
         onDataReceived(body);
+
+        if (body?.length <= 0) {
```

Comment:
dans quel cas body.length < 0 ?

Reply:
=== 0 si tu préfères, mais à priori pas de cas <= 0, c'est juste général

Reply:
j'ai jamais eu ce genre de reponse je demande à tout hasard. Si ca se trouve un 200 avec reponse vide peut arriver et on ne connait pas la signification => je suis plutot en faveur de body.length ===0

Reply:
There you go https://github.com/betagouv/pass-sport/pull/186/commits/0d66a6661ec257818d3698de02f42edaf8e4f8ee